sql-cli/sql_cli/macros/build_dag.py.jinja2

```diff
@@ -0,0 +1,29 @@
+{% macro build_dag(dag) %}
+with airflow.DAG(
+    dag_id="{{ dag.dag_id }}",
+    start_date=timezone.parse("{{ dag.start_date }}"),
+    {%- if dag.end_date %}
+    end_date=timezone.parse("{{ dag.end_date }}"),
+    {%- endif %}
+    catchup={{ dag.catchup }},
+    {% if dag.description %}
+    description="{{ dag.description }}",
+    {% endif %}
+    is_paused_upon_creation={{ dag.is_paused_upon_creation }},
+    orientation="{{ dag.orientation }}",
+    tags=[
+        {% for tag in dag.tags %}
+        "{{ tag }}",
+        {% endfor %}
+    ],
+    {% if dag.schedule.__class__.__name__ == "str" %}
+    schedule="{{ dag.schedule }}",
```

Comment:
Airflow < 2..4 doesn't have `schedule`, only `schedule_interval` - need to address this